google_chat_ros/CMakeLists.txt

```diff
@@ -41,15 +41,17 @@ catkin_generate_virtualenv(
 # euslisp
 file(GLOB EUSLISP_SCRIPTS scripts/*.l)
 install(FILES ${EUSLISP_SCRIPTS}
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-  )
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/scripts/)
 
 # python
 file(GLOB PYTHON_SCRIPTS scripts/*.py)
 catkin_install_python(
   PROGRAMS ${PYTHON_SCRIPTS}
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-  )
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/scripts/)
```

Comment:
This PR changes the installed directory from BIN to SHARE
@mqcmd196

Reply:
@k-okada 
I can understand that `*.l` libraries prefer to be on the same path for both `devel` and `install`. However, are Python files should be the same? They are resolved by `rosrun` if executable, and if the library, `catkin_python_setup`, `setup.py` do. I like to follow the common rule. Or for testing???

Reply:
That's an excellent question. I personaly do not like to install BIN directory, because `devel` and `install` should have same structure. Moreover, we've installed `scripts` and/or `nodes` under `share` directory using `install` CMake command, and it is unclear to me why we should relocate just to support Python3...
However, as an installing under BIN appears to be default behavior and I have changed my mind to fix all CMakeLists.txt, see 15c3e404
http://docs.ros.org/en/jade/api/catkin/html/howto/format2/installing_python.html

Reply:
> devel and install should have same structure.

Yes, I have the same opinion. `devel` seems to have made the development for ros users easier, but they had to care about the file structures in both `devel` and `install` especially when releasing the package. In my understanding, `ament` and `colcon` stopped using `devel` because of it, right...?
https://design.ros2.org/articles/ament.html#:~:text=package%20on%20Windows.-,Devel%20space,-catkin%20has%20the

Anyway, thank you for your update